templates/footer.hbs

```diff
@@ -1,22 +1,132 @@
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://help.getjobber.com/hc/en-us/categories/115001334107-Settings/">Settings</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://help.getjobber.com/hc/en-us/categories/115001334127-Billing-Add-ons/">Billing &amp; Integrations</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://help.getjobber.com/hc/en-us/categories/115001334067-The-Mobile-App/">Jobber App</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-column__inner">
+        <div class="footer-column__title">Resources</div>
+        <ul class="footer-list">
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://help.getjobber.com/hc/en-us/sections/115002368127">Jobber Glossary</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://help.getjobber.com/hc/en-us/categories/115001335727-Instructional-Videos">Instructional Videos</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://academy.getjobber.com/" target="_blank">Jobber Academy</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-column__inner">
+        <div class="footer-column__title">Jobber</div>
+        <ul class="footer-list">
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="https://getjobber.com" target="_blank">Back to Jobber</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer-column__inner">
+        <div class="footer-column__title">Contact Us</div>
+        <ul class="footer-list">
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="tel:+18887211115">1-888-721-1115</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="mailto:hello@getjobber.com">hello@getjobber.com</a>
+          </li>
+          <li class="footer-list__item">
+            <ul class="footer-social__list">
+              <li class="footer-social__item">
+                <a class="footer-social__link" href="https://www.facebook.com/GetJobber" target="_blank" rel="noopener noreferrer">
+                  <svg class="footer-social__icon footer-social__icon--facebook" viewBox="0 0 32 32" title="Jobber Facebook" xmlns="http://www.w3.org/2000/svg"><path d="M1.334 0C.597 0 0 .597 0 1.333v29.334C0 31.403.597 32 1.334 32H17V20h-4s-.889-3.889 0-5c.889-1.111 4 0 4 0v-4c0-4.133 2.617-6.167 6.304-6.167 1.766 0 3.284.132 3.726.19v4.32l-2.557.001c-2.005 0-2.473.953-2.473 2.351V15h5l-1 5h-4l.08 12h8.587c.736 0 1.333-.597 1.333-1.333V1.333C32 .597 31.403 0 30.667 0H1.334z" fill="#99A9B0" fill-rule="evenodd"/></svg>
```

Comment:
You have created the svg icons, but didnt use them here?

Reply:
![Screenshot 2019-12-17 11 13 44](https://user-images.githubusercontent.com/1953515/71022709-5dee6100-20be-11ea-8985-1bf147967e7d.png)

Just doesn't work, brah